### ISSUE TITLE
onClassName throws error on removal of node

### ISSUE THREAD

**Issue by RobAley** [josdejong/jsoneditor#657]
To reproduce, open the example "20_custom_css_style_for_nodes.html", and remove a node or two from one of the editors by clicking remove in the action menu. An error is thrown :

````
TypeError: this.childs is undefined[Learn More] jsoneditor.js:36541:22
	recursivelyUpdateCssClassesOnNodes jsoneditor-5.28.2/dist/jsoneditor.js:36541
	recursivelyUpdateCssClassesOnNodes jsoneditor-5.28.2/dist/jsoneditor.js:36542
	recursivelyUpdateCssClassesOnNodes jsoneditor-5.28.2/dist/jsoneditor.js:36542
	_onChange jsoneditor-5.28.2/dist/jsoneditor.js:30660
	_onAction jsoneditor-5.28.2/dist/jsoneditor.js:30593
	onRemove jsoneditor-5.28.2/dist/jsoneditor.js:39055
	onRemove jsoneditor-5.28.2/dist/jsoneditor.js:39031
	click jsoneditor-5.28.2/dist/jsoneditor.js:39968
	onclick jsoneditor-5.28.2/dist/jsoneditor.js:32819
````
I assume onClassName is trying to apply a class to the removed node.

**Comment by josdejong** [comment]
Thanks for reporting, turned out to be a simple bug in checking whether a property is not undefined. Fixed via https://github.com/josdejong/jsoneditor/commit/53a4f57dda947efd031aaab72a44f1c667d93b97, will publish the fix coming weekend.

**Comment by RobAley** [comment]
Thanks very much, that seems to work fine.

**Comment by josdejong** [comment]
The fix is now available in `v5.29.0`
